site/ja/xla/shapes.md

```diff
@@ -0,0 +1,113 @@
+# シェイプとレイアウト
+
+XLA `shape` proto（[xla_data.proto](https://www.tensorflow.org/code/tensorflow/compiler/xla/xla_data.proto)） は、N次元配列（略して配列）のランクとサイズ、データ型について記述します。
+
+
+## 用語、表記、決まりごと
+
+* 配列のランクは次元の数とおなじです。正しい配列のランクは、次元数が1以上の配列のことを指します。
+
+* 次元は、`N` 次元配列について、`0` から `N-1` で番号づけされます。次元番号は、便宜的につけられたラベルです。次元の順序は、シェイプのレイアウトの優先順序を意味しているわけではありません。
+
```

Comment:
[suggestion] Missing one sentence?

`The layout is determined by the Layout proto.`

Reply:
That's right! I missed this line.